src/api/pastel-rpc/transactions.ts

```diff
@@ -0,0 +1,170 @@
+import hex from 'hex-string'
+
```

Comment:
Please refactor this file and add tests, example: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/loading/__tests__/utils.test.ts. We don't want legacy code to be copied over without any modifications

Reply:
I have updated for this one